pytest.ini

```diff
@@ -2,5 +2,5 @@
 env =
     FLASK_ENV=unit_test
     FLASK_DEBUG=1
-    D:DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/postgres
+    D:DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/application_store
```

Comment:
please revert this

Reply:
...this change is causing tests to fail.

Reply:
my bad, removed